centos.org/ansible/provision.yml

```diff
@@ -8,40 +8,40 @@
       register: cico_get_data
       retries: 2
       delay: 310
-      until: cico_get_data.results is defined
+      until: cico_get_data.ansible_module_results is defined
```

Comment:
this will need another change once https://github.com/CentOS/python-cicoclient/commit/d7cda54e269e7cfd5eee156d1f6b0463d3b4d861 is released, but then it should be stable